service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -148,6 +166,13 @@ public String getMostRecentUploadedOrderDocumentUrl(final List<Element<Generated
             .getDocument().getBinaryUrl();
     }
 
+    public void removeOrderProperties(CaseDetails caseDetails) {
+        caseDetails.getData().remove("orderTypeAndDocument");
```

Comment:
Could be done with `caseDetails.getData().keySet().removeAll(List.of(...`, but not sure if that's better, so happy to leave it as it is.